cookbook/form/data_transformers.rst

```diff
@@ -63,27 +57,24 @@ class::
 
     use Symfony\Component\Form\CallbackTransformer;
     use Symfony\Component\Form\FormBuilderInterface;
-    use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
     // ...
 
     class TaskType extends AbstractType
     {
         public function buildForm(FormBuilderInterface $builder, array $options)
         {
-            $builder->add('description', TextareaType::class);
+            $builder->add('labels', TextType::class);
 
-            $builder->get('description')
+            $builder->get('labels')
                 ->addModelTransformer(new CallbackTransformer(
-                    // transform <br/> to \n so the textarea reads easier
-                    function ($originalDescription) {
-                        return preg_replace('#<br\s*/?>#i', "\n", $originalDescription);
+                    // transform array to string so the input reads easier
+                    function ($originalLabels) {
```

Comment:
What if we rename these variables:

```
$originalLabels  -> $labelsAsArray
$submittedLabels -> $labelsAsString
```

This is just a proposal. Please don't make any change until our doc managers approve/reject this idea. Thanks!